src/handlers/event.handler.ts

```
@@ -0,0 +1,68 @@
+      maxAttendees: data.eventDetailsData.maxAttendees,
+      recordings: data.eventDetailsData.recordings,
+      status: data.eventDetailsData.status,
+      description: data.eventDetailsData.description,
+      meetingDetails: data.eventDetailsData.meetingDetails,
+      createdBy: data.eventDetailsData.createdBy,
+      updatedBy: data.eventDetailsData.updatedBy,
+      idealTime: data.eventDetailsData.idealTime,
+      metadata: data.eventDetailsData.metadata,
+      attendees: data.eventDetailsData.attendees,
+    };
+    const savedEventDetails = await this.dbService.saveEventDetailsData(eventDetailsData);
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)